spotify_confidence/analysis/frequentist/confidence_computers/generic_computer.py

```diff
@@ -918,10 +918,10 @@ def _add_adjusted_p_and_is_significant(df: DataFrame, arg_dict: Dict) -> DataFra
         return df
 
     def _compute_sequential_adjusted_alpha(df: DataFrame, method_column: str, arg_dict: Dict) -> Series:
-        if all(df[method_column] == "z-test"):
-            return confidence_computers["z-test"].compute_sequential_adjusted_alpha(df, arg_dict)
+        if df[method_column].isin([ZTEST, ZTESTLINREG]).all():
+            return confidence_computers[ZTEST].compute_sequential_adjusted_alpha(df, arg_dict)
```

Comment:
So we don't need to use the ztestlinreg computer here?

Reply:
No, the idea is that with variance reduction we've done the adjustment, and now we can just use the usual ZTEST stuff. This is what we used to inherit.